public/app/features/commandPalette/values.ts

```diff
@@ -0,0 +1,4 @@
+export const RECENT_DASHBOARDS_PRORITY = 4;
+export const DEFAULT_PRIORITY = 3;
+export const PREFERENCES_PRIORITY = 2;
+export const SEARCH_RESULTS_PRORITY = 1; // Dynamic actions should be below static ones so the list doesn't 'jump' when they come in
```

Comment:
this comment doesn't seem to be correct. if i search for `new`, i see `Dashboards` then `actions` then `Recently viewed dashboards` 
![image](https://user-images.githubusercontent.com/20999846/211557521-bdae96d6-8dd6-495a-aa4d-ceb970d132cc.png)

don't feel super strongly about this right now as i suspect it's gonna change following our mob session/outcome of https://github.com/grafana/grafana/issues/61093 anyway

Reply:
Ahhh, you're right. I think the Section needs priority instead, which impacts the types i made, so I'll tackle this seperately.